indexer/CHANGELOG.md

```diff
@@ -0,0 +1,272 @@
+# Changelog
+
+## 0.11.0
```

Comment:
I've opted to keep the `near/near-indexer-for-explorer` `CHANGELOG.md` and add the lake migration as an entry on top. The migration seems more like a progression from the current indexer so this made more sense to me.

Reply:
BTW, please note that we used to write the section `Breaking changes` describing what exactly is breaking. Could you try to write one here as well, please?

Reply:
Updated to the following:
```md
## 0.11.0

* Migrate to NEAR Lake Framework, `indexer-explorer` no longer runs a NEAR node and instead consumes block from [near-lake-indexer](https://github.com/near/near-lake-indexer).

### Breaking Changes

* Storing genesis file is no longer possible
* Command line arguments for starting the indexer have changed slightly, see [Readme](../README.md) for updated reference
* AWS credentials are now required for reading data from S3
```

Let me know if I've missed any breaking changes

Reply:
Please refer the issue to the point:

> Storing genesis file is no longer possible [issue_link]

just for the sake

Reply:
Good call 👍🏽

Reply:
Good call 👍🏽